operator/src/test/java/org/keycloak/operator/ClusteringE2EIT.java

```diff
@@ -118,6 +118,8 @@ public void testKeycloakCacheIsConnected() throws Exception {
                     Log.info("Checking Keycloak is stable.");
                     CRAssert.assertKeycloakStatusCondition(crSelector.get(), KeycloakStatusCondition.READY, true);
                 });
+        // Remove the completed pod for the job
+        realmImportSelector.delete();
```

Comment:
Out of curiosity, why is this change necessary?

Reply:
Good question!

 - By applying the entire template the Pod of the Job is inheriting more labels
 - we are (a little below in this test) testing the connection toward each single Pods selected by labels
 - now the Job Pod is included in the selection but it's in "Completed" state (e.g. not running)
 
 Solution:
  - removing the realm import CR deletes all the connected resources (and the Pod too)

Reply:
Thank you for clarifying!